build/test.Dockerfile

```diff
@@ -10,6 +10,9 @@ COPY --from=builder /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/
 COPY $TEST_NAME /test
 
 LABEL name="botkube" \
-    test=$TEST_NAME
+      test=$TEST_NAME \
+      org.opencontainers.image.source="git@github.com:kubeshop/botkube.git" \
+      org.opencontainers.image.title="BotKube E2 tests" \
+      org.opencontainers.image.description="BotKube E2E tests which are run against BotKube installed on Kubernetes cluster and Slack API."
```

Comment:
Do we want here the `documentation` label as well? I would put it, because why not 🤔 
```
      org.opencontainers.image.documentation="https://botkube.io"
```

Reply:
it's about e2e test I'm not sure if this should be mentioned here, we have it in BotKube image